de.dlr.sc.virsat.docs.feature/src/docs/VirSat_Core_Developer_Manual.adoc

```diff
@@ -160,12 +160,255 @@ Congratulations. You have successfully set up your Git integration!
 
 == Generic Systems Engineering (Concept) Language 
 
-=== Categories and Properties
+Concepts are described using the Generic Systems Engineering Language (GSEL).
+The following gives a break-down on the key words and functionalities provided by the GSEL.
+
+=== Concepts
+
+A  _Concept_ is always the root of a concept.concept file.
+It serves as the main container of the concept.
+It also fulfills the task of carrying global meta-information.
+
+A Concept may also have the following attributes:
+
+displayname::
+This name will be shown to the user when interacting with the concept in the UI.
+For example when activating a concept, the displayname is shown.
+
+description::
+The description of the concept.
+
+version:: 
+The version number of the concept. 
+Must be given in the form of `.` separated numbers.
+For each version number, the IDE tools generate a corresponding `concept_vVERSION.xmi` in the concept folder.
+This file is used to automtically migrate old repositories based on an older concept version to the newest version.
+
+WARNING: When editing a concept, the version number should be increased before making any changes to the concept.
+Otherwise the changes will be considered to be part of the old version.
+Not doing this will cause automatic migration to fail!
 
 === Structural Elements
 
+_Structural Elements_ decompose a system into smaller components.
+For example, in the _CEF_ concept, a system is structurally decomposed into _System_ -> _SubSystem_ -> _Equipment_.
+Structural Elements are also the containers for data information (Categories and Properties).
+To support configuration management and to bundle common data, Structural Elements can also inherit data from one another.
+The instance of a Structural Element is called a _Structural Element Instance_.
+
+A Structural Element can have the following additional attributes:
+

```

Comment:
Here are also not all attributes listed:
e.g. short name 
name etc.

Reply:
Nice finde, forgot about them. Added name and shortname for Category and Structural Element.